p9-presentation/sections/_experiments.qmd

```diff
@@ -1 +1,347 @@
+:::
+::: {.fragment .fade-up}
+- Time is a contraint - which experiments are most meaningful?
+:::
+</div>
+::: 
+
+::: notes
+**Experiments introduction**
+<br>
+Once the replication process was done we had to decide on a set of experiments that would best answer our problem definition.
+<br>
+A tricky aspects of answering the problem definition is defining what constitutes a component and in turn, how to define an experiment that would allow us to determine its contribution to the over all error.
+<br>
+On top of this, was the looming time constraint for delivery on the project. This meant that we had to be selective on the type of experiments that we chose.
+We had to ensure that the experiments were impactful enough to be able to determine something about the pipeline.
+:::
+
+## Our Choice of Experiments
+<div class="experiments-spacing">
+<p class="experiments-header">Outlier based experiments:</p>
+</div>
+
+::: {.fragment .fade-up}
+1. Evaluating the necessity of automated outlier removal in the PLS1-SM
+:::
+::: {.fragment .fade-up}
+::: {style="font-size: 0.5em;"}
+| Element | Baseline | Without outlier removal | PLS1-SM (Clegg) |
+|---------|---------:|------------------------:|----------------:|
+| SiO2    |     5.81 |                    5.81 |            4.33 |
+| TiO2    |     0.47 |                    0.47 |            0.94 |
+| Al2O3   |     1.94 |                    1.91 |            2.85 |
+| FeOT    |     4.35 |                    4.35 |            2.01 |
+| MgO     |     1.17 |                    1.17 |            1.06 |
+| CaO     |     1.43 |                    1.44 |            2.65 |
+| Na2O    |     0.66 |                    0.67 |            0.62 |
+| K2O     |     0.72 |                    0.70 |            0.72 |
+
+
+:::
+:::
+
+::: {.notes}
+**The experiments that we chose**
+<br>
+We settled on conducting 5 sets of experiments. 
+The first set of experiments were centered around outlier removal.
+In the initial stages of our implementation we realized that we would not be able to do outlier detection in the same way that Clegg et. al did it. 
+Both in terms of the PLS-SM side of the pipeline and also in the ICA side of the pipeline.
+Therefore we thought it would be interesting to conduct experiments that tested different aspects of our outlier removal process.
+
+**(1) Automated outlier removal**
+<br>
+The first experiment that we decided to conduct was to evaluate the effects of not doing outlier removal vs. our automated outlier removal for PLS-SM. 
+The outcome of this experiment would provide us answers to two things. The first being an evaluation of our automated outlier removal. 
+The second being an implicit evaluation of the PLS-SM model to handling of outliers. 
+The implicit evaluation of the PLS-SM is interesting since if we did not see a big in RMSE between our own and clegg et. al's, then that would call into question of doing outlier removal at all for PLS-SM.
```

Comment:
```suggestion
The implicit evaluation of the PLS-SM is interesting since if we did not see a big difference in RMSE between our own and clegg et. al's, then that would call into question of doing outlier removal at all for PLS-SM.
```